web_src/css/user.css

```diff
@@ -26,9 +26,14 @@
 }
 
 .user.profile .ui.card .extra.content > ul > li:not(:last-child) {
+  display: flex;
   border-bottom: 1px solid var(--color-secondary);
 }
 
+.user.profile .ui.card .extra.content > ul > li:not(:last-child) > :nth-child(2) {
+  flex-grow: 1;
```

Comment:
Using "flex" layout is good while the "ui right" (float: right) should be avoided.

Reply:
"flex" layout makes "ui right" redundant anyways, removed it everywhere.